activerecord/test/cases/base_test.rb

```diff
@@ -1464,4 +1464,8 @@ def test_default_values_are_deeply_dupped
     assert SubDeveloper.new.respond_to?(:last_name=)
     assert SubDeveloper.new.respond_to?(:last_name?)
   end
+
+  test "ignored columns are stored as a array of string" do
+    assert_equal(SymbolIgnoredDeveloper.ignored_columns, %w(first_name last_name))
```

Comment:
Can you place expected value in the left-hand side?

```
% git grep -n 'assert_equal expected' | tail
test/cases/scoping/default_scoping_test.rb:212:    assert_equal expected_2, received_2
test/cases/scoping/default_scoping_test.rb:218:    assert_equal expected, received
test/cases/scoping/default_scoping_test.rb:224:    assert_equal expected, received
test/cases/scoping/default_scoping_test.rb:230:    assert_equal expected, received
test/cases/scoping/default_scoping_test.rb:236:    assert_equal expected, received
test/cases/scoping/default_scoping_test.rb:246:    assert_equal expected, received
test/cases/scoping/default_scoping_test.rb:300:    assert_equal expected, received
test/cases/scoping/named_scoping_test.rb:121:    assert_equal expected.sort_by(&:id), authors(:david).posts.containing_the_letter_a.sort_by(&:id)
test/cases/scoping/named_scoping_test.rb:134:    assert_equal expected.sort_by(&:id), authors(:david).comments.containing_the_letter_e.sort_by(&:id)
test/cases/serialized_attribute_test.rb:256:    assert_equal expected, error.to_s
```

Reply:
Sorry.. fixd.